napari/_vispy/markers.py

```diff
@@ -0,0 +1,17 @@
+from vispy.scene.visuals import Markers as BaseMarkers
+
+
+class Markers(BaseMarkers):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def _compute_bounds(self, axis, view):
+        if self._data is None:
+            return None
+        pos = self._data['a_position']
+        if pos is None:
+            return None
+        if pos.shape[1] > axis:
+            return (pos[:, axis].min(), pos[:, axis].max())
+        else:
+            return (0, 0)
```

Comment:
Can you explain the need for this class in a comment or docstring?

Reply:
Comment added, I'll probably make a vispy PR soon to fix this, but haven't done so yet